tests/checks/integration/test_nginx.py

```diff
@@ -22,27 +22,42 @@ def setUp(self):
                 {
                     'nginx_status_url': 'http://localhost:44441/nginx_status/',
                     'tags': ['first_one'],
+                    'nginx_ssl_verify': True,
                 },
                 {
                     'nginx_status_url': 'http://dummyurl:44441/nginx_status/',
                     'tags': ['dummy'],
+                    'nginx_ssl_verify': True,
                 },
                 {
                     'nginx_status_url': 'http://localhost:44441/nginx_status/',
                     'tags': ['second'],
+                    'nginx_ssl_verify': True,
```

Comment:
Yeah, actually I referenced that in my comment above in this issue.

I basically confirm that this doesn't actually test functionality with the verify option for requests.get(), and that to do that would require an https endpoint on localhost to run a request against. However this is testing the requests submodule, and not our code, so I think an ssl_verify check here is unnecessary. The important part is that our config gets picked up.

But, I'm all for total coverage, so if you think adding this explicitly to the test is wise, then that's no problem. Is there an existing https endpoint on localhost that the test suite can access?